src/Files.App/Services/PreviewPopupProviders/QuickLookProvider.cs

```diff
@@ -45,6 +45,10 @@ private async Task DoPreviewAsync(string path, string message)
 			{
 				client.Close();
 			}
+			catch (IOException)
+			{
+				client.Close();
```

Comment:
Although not directly related to this fix, `client` is declared with `using`, so there is no need to explicitly call `Close()`.

Reply:
That's good to know